src/pydata_sphinx_theme/__init__.py

```diff
@@ -925,38 +925,50 @@ def run(self, **kwargs):
                 self.platform = self.supported_platform.get(uri.netloc)
                 if self.platform is not None:
                     node.attributes["classes"].append(self.platform)
-                    node.children[0] = nodes.Text(self.parse_url(uri.path))
+                    node.children[0] = nodes.Text(self.parse_url(uri))
 
-    def parse_url(self, path):
+    def parse_url(self, uri):
         """
         parse the content of the url with respect to the selected platform
         """
+        path = uri.path
 
-        # split the url content
-        # be careful the first one is a "/"
-        parts = path.split("/")
+        if path == "":
+            # plain url passed, return platform only
+            return self.platform
+
+        # the leading "/" is removed
```

Comment:
Rather than just saying the / is removed, how about a few words about why it is removed?

Reply:
good point, will add that!